reaper/index.js

```diff
@@ -27,11 +27,15 @@ exports.handler = (event, context, callback) => {
                         fetch(image.uri, { headers, method: 'DELETE' })
                         .then(res => {
                             if(res.status >= 200 && res.status < 300) {
-                                logger.log( `${res.status}: successfully deleted picture ${imageId}`);
+                                logger.log( `${res.status}: successfully deleted picture ${imageId}`, {
```

Comment:
this is how we add arbitrary markers: https://github.com/guardian/elk-kinesis-logger/blob/master/examples/advanced-example.js#L17-L19